chicagobub/scripts/bubble_chart.js

```diff
@@ -28,11 +27,14 @@ function bubbleChart() {
   var bubbles = null;
   var nodes = [];
 
+  var yearToSplitOn = null;
```

Comment:
Here we have a global variable to store which year-data we want the bubbles to move towards.